tests/metrics/network/network-metrics-cpu-consumption.sh.in

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures cpu % consumption using an inter (docker<->docker) 
+#  network bandwidth using iperf2
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Port number where the server will run
+port=5001:5001
+# Using this image as iperf is not working
+# see (https://github.com/01org/cc-oci-runtime/issues/152)
+# Image name
+image=gabyct/network
```

Comment:
@sboeuf so the solution to have network metrics was that I created an image with all dependencies, actually this image is already part of other metrics scripts that are part of our repo